hack/import-flux2-assets.sh

```diff
@@ -157,10 +157,10 @@ gen_crd_doc() {
   TMP_METADATA="$TMP/flux.json"
   TMP_BIN="$TMP/flux.tar.gz"
 
-  curl -o "${TMP_METADATA}" --retry 3 -sSfL "https://api.github.com/repos/fluxcd/flux2/releases/latest"
+  curl -u fluxcdbot:$GITHUB_TOKEN -o "${TMP_METADATA}" --retry 3 -sSfL "https://api.github.com/repos/fluxcd/flux2/releases/latest"
```

Comment:
Can the `fluxcdbot` be moved to a variable? This would help e.g. running the script locally.